.github/workflows/code-quality.yml

```diff
@@ -100,6 +100,12 @@ jobs:
           clang-format --version
           cpplint --version
 
+      - name: CMake lint
+        shell: bash
+        run: |
+          python -m pip install cmakelang pyyaml
+          find . \( -name "CMakeLists.txt" -o -wholename "*/cmake/*.cmake" \) | xargs cmake-lint
```

Comment:
[consider][minor] Might be good to exclude hidden (i.e. `.`-prefixed) files, e.g. from [askubuntu](https://askubuntu.com/a/1127030)
```
find <start-point> -path '*/.*' -prune -o <expression> -print
```

Reply:
Yeah good idea.

Couldn't quite get that formulation working, but went with 
```
find . -not -path '*/.*' -a -name "CMakeLists.txt" -o -not -path '*/.*' -name "*.cmake"
 ```
which I think is okay.

https://github.com/OpenAssetIO/OpenAssetIO/pull/811/commits/c37e253dcc71eb7d0c1f9c5c67b6a2002b21aa5e